scapy/arch/linux.py

```diff
@@ -22,6 +23,7 @@
 import scapy.arch
 from scapy.error import warning, Scapy_Exception, log_interactive, log_loading
 from scapy.arch.common import get_if
+from scapy.modules.six.moves import map
```

Comment:
Of course it *does* exist, it's just been made lazy instead of greedy, as most builtin things returning iterables. It now returns a generator, which does not have to be `list()`ed when using the star operator.

```python
>>> list(map(lambda x: x * 2, [1, 2, 3]))
[4, 5, 6]
>>> def foo(a, b, c):
...    print(a, b, c)
>>> foo(*map(lambda x: x * 2, [1, 2, 3]))
4 5 6
```